lib/components/form/SiteDetails.tsx

```diff
@@ -65,14 +62,10 @@ const SiteDetails: FC<SiteDetailsProps> = ({ site }) => {
 
   return (
     <div className="w-full md:flex-col md:justify-center">
-      <div className="flex h-[var(--nav-height)] w-full flex-row items-center justify-between bg-ocf-black px-5 md:justify-center md:py-2">
-        <div className="md:hidden">
-          {!(page === Page.Location && sites.length === 0) && (
-            <BackButton onClick={lastPageCallback} />
-          )}
-        </div>
-        <NowcastingLogo />
-      </div>
```

Comment:
Is this stuff also supposed to be removed?

Reply:
Yep, replaced by `<BackNav>`